client/influxdb_test.go

```diff
@@ -547,3 +577,33 @@ func TestClient_NoTimeout(t *testing.T) {
 		t.Fatalf("unexpected error.  expected %v, actual %v", nil, err)
 	}
 }
+
+func testClient_WriteUint64(t *testing.T) {
```

Comment:
Yes! how strange, first commit had it capitalised, must have been a mistaken keypress.